### PR TITLE
GNOME - Add missing `Ctrl+Q` shortcut

### PR DESCRIPTION
**Pull request by DaPigGuy**
Again :upside_down_face:

### CODE REVIEW

File: NickvisionMoney.GNOME/Views/MainWindow.cs [hunk 1]
```diff
@@ -490,6 +495,16 @@ private void KeyboardShortcuts(Gio.SimpleAction sender, EventArgs e)
         shortcutsDialog.Show();
     }
 
+    /// <summary>
+    /// Occurs when quit action is triggered
+    /// </summary>
+    /// <param name="sender">Gio.SimpleAction</param>
+    /// <param name="e">EventArgs</param>
+    private void Quit(Gio.SimpleAction sender, EventArgs e)
```
Comment (nlogozzo) [inline NickvisionMoney.GNOME/Views/MainWindow.cs:503]: Single-lined methods in C# are usually condensed into one line. Could you make this:
```csharp
private void Quit(Gio.SimpleAction sender, EventArgs e) => _application.Quit();
```

**Review by nlogozzo — CHANGES_REQUESTED** [review]

File: NickvisionMoney.GNOME/Views/ShortcutsDialog.cs [hunk 2]
```diff
@@ -100,6 +100,12 @@ public ShortcutsDialog(Localizer localizer, Gtk.Window parent)
                                     <property name='accelerator'>&lt;Control&gt;question</property>
                                 </object>
                             </child>
+                            <child>
+                                <object class='GtkShortcutsShortcut'>
+                                    <property name='title'>{localizer["Quit"]}</property>
+                                    <property name='accelerator'>&lt;Control&gt;q</property>
+                                </object>
+                            </child>
```
Comment (fsobolev) [inline NickvisionMoney.GNOME/Views/ShortcutsDialog.cs:108]: I think it should be below help shortcut

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionMoney.GNOME/Views/MainWindow.cs (hunk 1 above)
Comment (DaPigGuy) [reply NickvisionMoney.GNOME/Views/MainWindow.cs]: Not all that familiar with C#, TIL :slightly_smiling_face:

File: NickvisionMoney.GNOME/Views/ShortcutsDialog.cs (hunk 2 above)
Comment (DaPigGuy) [reply NickvisionMoney.GNOME/Views/ShortcutsDialog.cs]: Done :+1:

**Review by fsobolev — APPROVED** [review]

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**